ports/z3/vcpkg.json

```
@@ -1,6 +1,6 @@
 {
   "name": "z3",
-  "version-semver": "4.8.10",
+  "version-semver": "4.8.12",
```

```suggestion
  "version-string": "4.8.12",
```
This might be `version-string` instead of `version-semver`. Since it also has the prefix 'z3-' for the version.Please refer to this [doc](https://github.com/microsoft/vcpkg/blob/master/docs/users/versioning.md).

---

Judging by https://github.com/Z3Prover/z3/blob/master/RELEASE_NOTES, I agree that z3 does not seem to follow semver. But shouldn't we switch to `version` instead of `version-string`? After all, the version is still `4.8.12` and not `z3-4.8.12`.

---

Thanks for your details. Seems `version `is more proper here.

---

I switched to `version`.